### DISCUSSION TITLE
How do you get back to the database view

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#496] [Q&A] [answered]
When I open the page in markdown I can edit things but I can't find a button to load the database back up. 

Is there a button/where is it, or do I need to switch pages to reload the database view?

**Comment by RafaelGB** [comment, marked as answer]
With Obsidian .16.3 there are back/forward buttons per tab natively

**Reply by DannyHatcher-Git** [reply]
But that doesn't change the markdown file to the database view, it just moves back a page.

**Reply by RafaelGB** [reply]
oh I got it now. You need to close the file from dots menu and reopen the note

**Reply by DannyHatcher-Git** [reply]
Ok, that is what I have been doing. Just wondering if there was another way. Thanks 😁

**Reply by DominicD** [reply]
This should be pinned 👍

**Reply by huyz** [reply]
Obsidian Annotator has a command "Toggle Annotation/Markdown Mode". Perhaps DB Folder needs something similar.

**Reply by Lmasey** [reply]
This could be user error, but I have closed the database and even restarted Obsidian and I can't fix it!
